local_storage_utils.ts

```diff
@@ -1,13 +1,8 @@
-import {Recipe} from "./types";
-
 export const setToLocalStorage = (key: string, data: any) => {
     localStorage.setItem(key, JSON.stringify(data))
 }
 
-export const getFromLocalStorage = (key: string): Array<string> => {
+export function getFromLocalStorage<Type>  (key: string): Type  {
```

Comment:
no function, use const,
we prefer `T`.
Also, you should cast JSON.parse to your generic type